common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -680,14 +686,19 @@ def convert(content, input_format, output_format):
                     content_str = content.decode('latin-1')
             else:
                 content_str = content
-
+            try:
+                content_dict = json.loads(content_str)
+            except Exception:
+                log.exception(
+                    "Failed to convert {input_format} to {output_format} for {content_str}."
```

Comment:
Is there a way to put just a pointer to the location of the source of the `content_str` instead? The worry I have here is that there will be a malformed (or maliciously formed) 100 MB transcript file that will end up destroying our logging.

Reply:
Ah great idea. I will look into this, definitely one exists

Reply:
The annoying thing about this being a util is that it just takes in content and input & output formats. Maybe just take the first ___ characters of the content? where ___ is some number.

Reply:
Or use traceback to find caller method?

Reply:
Taking the first n bytes is fine (100? 1000?). Just two requests there:

1. Please escape new lines.
2. Please verify that this doesn't blow up when it encounters non-Latin text. I'm not sure what the string interpolation rules are here.

Actually, come to think of it, don't you need an `f` prefix on this string to actually interpolate the values?

Reply:
Will do all three.

Reply:
I think I have handled all non-latin text by running `repr()` on the new line-removed string before it is handed to the logger.